classes/01class/exercises/c01-e01/c01-e01.txt

```diff
@@ -0,0 +1,11 @@
+✦ 👽 henriquediniz aws --version
+aws-cli/1.17.5 Python/2.7.16 Darwin/19.2.0 botocore/1.14.5
+academy/classes/01class/Exercises/c01-e01 on  henriqueediniz-c01-e01 [?] 
+✦ 👽 henriquediniz docker -v
+Docker version 19.03.5, build 633a0ea
+academy/classes/01class/Exercises/c01-e01 on  henriqueediniz-c01-e01 [?] 
+✦ 👽 henriquediniz terraform version
+Terraform v0.12.19
+
+Your version of Terraform is out of date! The latest version
```

Comment:
Do you have a specific reason for not running the latest version? If no, can you update terraform and post the new version here?

Reply:
No, I already update now.

Reply:
cool, can you update this file with the new version?

Reply:
I updated now. You can check in this link https://github.com/devopsacademyau/academy/pull/21/commits/28a9d0648c37ed36f42808f7782c502911795693